NFIQ2/NFIQ2Algorithm/include/nfiq2.hpp

```diff
@@ -21,4 +21,11 @@
 #include <nfiq2_timer.hpp>
 #include <nfiq2_version.hpp>
 
+/**
+ * 	This namespace contains classes and functions that
+ * 	contribute to the calculation of NFIQ2 scores.
+ */
```

Comment:
In general, don't use "This _X_ does..."

```suggestion
/** Classes and functions that contribute to the calculation of NFIQ2 scores. */
```